common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -68,33 +68,40 @@ export function sourcemapsExtra() {
  */
 export type WarningInhibitor = (warning: RollupWarning) => boolean;
 
+function matchesPathSegments(str: string | undefined, segments: string[]): boolean {
+  return (
+    str?.includes(segments.join(path.sep)) ||
+    // on Windows in powershell, path.sep is "\\" but reported warnings use "/"
+    str?.includes(segments.join("/")) === true
```

Comment:
perhaps this is equivalent and more concise:
```suggestion
    str?.includes(segments.join("/"))
```

Reply:
It was needed to convert possible undefined to boolean. However, I found that "/" is always used on Linux and Windows, presumably on MacOS too, so I changed the check.